corehq/apps/reports/static/reports/ko/saved_reports.js

```diff
@@ -326,6 +326,6 @@ $.fn.reportUserConfigurableConfigEditor = function (options) {
 $.fn.reportConfigList = function (options) {
     this.each(function(i, v) {
         var viewModel = new ReportConfigsViewModel(options);
-        ko.applyBindings(viewModel, $(this).get(i));
+        $(this).koApplyBindings(viewModel);
```

Comment:
Definitely feel free to simplify to exclude cases that you're sure never happen! Much better to let someone reimplement logic when they need it than to preserve logic hoping it might one day be used.